neurots/generate/algorithms/tmdgrower.py

```diff
@@ -45,23 +44,36 @@ class TMDAlgo(AbstractAlgo):
     """
 
     def __init__(
-        self, input_data, params, start_point, context=None, random_generator=np.random, **_
+        self,
+        input_data,
+        params,
+        start_point,
+        skip_validation=False,
```

Comment:
we need to pass it via NeuronGrower, which is the class used in region-grower, otherwise we won't have access to this parameter

Reply:
This is a tmd-algo specific test (this value does not exist in other distributions except for the tmd ones). However, you can pass it down when you define the NeuronGrower, I am adding a test for that to see how it works (and increase coverage).